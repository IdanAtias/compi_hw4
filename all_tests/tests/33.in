{
F x;
F y;
{F y=100 F;}
F z;
}
