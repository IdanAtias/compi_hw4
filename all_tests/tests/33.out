line 4: variable y is already defined
